gordo/machine/model/models.py

```diff
@@ -78,11 +76,22 @@ def __init__(
             building function and/or any additional args to be passed
             to Keras' fit() method
         """
-        self.build_fn = None
-        self.history = None
-
         self.kind = self.load_kind(kind)
         self.kwargs: Dict[str, Any] = kwargs
+        self._history = None
+
+        # This new keras wrapper expects most of these kwargs to be set to the model attributes and uses them for
+        # defaults in some places, but always gives precedence to kwargs passed to respective fit, predict and compile
+        # methods, so this is just to make it happy again
+        _expected_kwargs = {
+            *self._fit_kwargs,
+            *self._predict_kwargs,
+            *self._compile_kwargs,
```

Comment:
```suggestion
            *KerasRegressor._fit_kwargs,
            *KerasRegressor._predict_kwargs,
            *KerasRegressor._compile_kwargs,
```
in case of self not clear what the exact source of these constants